editor/plugins/gradient_editor_plugin.h

```diff
@@ -33,25 +33,106 @@
 
 #include "editor/editor_inspector.h"
 #include "editor/editor_plugin.h"
-#include "gradient_editor.h"
+#include "scene/resources/gradient_texture.h"
```

Comment:
The include is redundant here since you forward declare the class. Has to be moved to the cpp file.